File: frontend/src/features/actividades/asignar_escritura/models/asignar_escritura.ts
export interface AsignarEscritura {
  id_asignar_escritura: number;
  id_expediente: number;
  id_actividad: string;
  abogado: string;
  observaciones: string;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/asignar_escritura.cs
using Common.Domain.Models;

namespace Multibanca.Domain.Models.Multibanca
{
    public class asignar_escritura : base_auditoria
    {
        public int id_asignar_escritura { get; set; }
        public long id_expediente { get; set; }
        public string id_actividad { get; set; } = string.Empty;
        public string abogado { get; set; } = string.Empty;
        public string observaciones { get; set; } = string.Empty;
    }
}
